### DISCUSSION TITLE
does vector support containerd runtime?

### DISCUSSION THREAD

**Discussion by lddlww** [vectordotdev/vector#21811] [Q&A]
i found the parameter docker_host  can't compatible containerd runtime，in the docker_logs source; when i set docker_host="/var/run/containerd/containerd.sock"， vector through follow error：
```
2024-11-15T06:15:39.065760Z ERROR source{component_kind="source" component_id=dummy_logs component_type=docker_logs}: vector::sources::docker_logs: Listing currently running containers failed. error=Error in the hyper legacy client: client error (SendRequest)
```

am i missing something?

**Comment by pront** [comment]
Hi @lddlww, this looks like a permissions issue. Did you pass `-v /var/run/docker.sock:/var/run/docker.sock` or something similar to your `docker run`?

**Comment by lddlww** [comment]
no，i am not start with docker instead of start containerd standalone，so there is not docker.sock

**Reply by pront** [reply]
Just a guess, but this old thread might be relevant here: https://github.com/vectordotdev/vector/issues/5410#issuecomment-739138393
